inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/MetaSink.java

```diff
@@ -796,6 +802,14 @@ public void update() {
                 Runtime.getRuntime().availableProcessors() + 1);
     }
 
+    private Map getNewDimension(String otherKey, String value) {
+        Map dimensions = new HashMap<>();
```

Comment:
Sink.process is running in one thread, dimensions can be one object.

Reply:
different topic need use different dimension.

Reply:
Before invoking metric operation, it is enough to put curent topic to dimensions.

dimensions.put(DataProxyMetricItem.KEY_SINK_DATA_ID, event.getHeaders().get(TOPIC));
DataProxyMetricItem metricItem = this.metricItemSet.findMetricItem(dimensions);
metricItem.readSuccessCount.incrementAndGet();
metricItem.readFailSize.addAndGet(event.getBody().length);

Reply:
here every topic need use itself dimension obj